torchmetrics/functional/text/bleu.py

```diff
@@ -139,38 +139,51 @@ def _bleu_score_compute(
 
     log_precision_scores = tensor([1.0 / n_gram] * n_gram, device=device) * torch.log(precision_scores)
     geometric_mean = torch.exp(torch.sum(log_precision_scores))
-    brevity_penalty = tensor(1.0, device=device) if trans_len > ref_len else torch.exp(1 - (ref_len / trans_len))
+    brevity_penalty = tensor(1.0, device=device) if preds_len > target_len else torch.exp(1 - (target_len / preds_len))
     bleu = brevity_penalty * geometric_mean
 
     return bleu
 
 
+@deprecated(
+    args_mapping={"translate_corpus": "preds", "reference_corpus": "target"},
+    target=True,
+    stream=logging.warning,
+    deprecated_in="0.7",
+    remove_in="0.8",
+)
 def bleu_score(
-    translate_corpus: Union[str, Sequence[str]],
-    reference_corpus: Sequence[Union[str, Sequence[str]]],
+    preds: Union[None, str, Sequence[str]] = None,
+    target: Union[None, Sequence[Union[str, Sequence[str]]]] = None,
     n_gram: int = 4,
     smooth: bool = False,
 ) -> Tensor:
     """Calculate `BLEU score`_ of machine translated text with one or more references.
 
     Args:
-        translate_corpus:
+        preds:
             An iterable of machine translated corpus
-        reference_corpus:
+        target:
             An iterable of iterables of reference corpus
         n_gram:
             Gram value ranged from 1 to 4 (Default 4)
         smooth:
             Whether or not to apply smoothing – see [2]
+        translate_corpus:
+            .. deprecated:: v0.7
+                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
+        reference_corpus:
+            .. deprecated:: v0.7
+                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
```

Comment:
drop these if they are not in the func signature...

Reply:
@Borda Ok, I'll remove these. And do you think we will go with `pyDeprecate` then? :]

cc: @SkafteNicki

Reply:
Agree that we can use `pyDeprecate` temporary for this